app/ceom/aboutus/views.py

```diff
@@ -48,8 +50,63 @@ def group_photos(request, selYear = None):
         'available_years' : available_years,
         'photos': photos,
     })
+
+def user_stats(request):
+    data = {}
+    request_list = request.GET.getlist('p')
+    type = request_list[0]
+    if (len(request_list) == 1):
+        year_min = 2007
+        year_max = date.today().year
+    else: 
+        year_min = int(request_list[1]) 
+        year_max = int(request_list[2])  
+    data['x_values'] = list(range(year_min, year_max+1))
+    data['year_range'] = range(2007, date.today().year+1)
+    data['type'] = type
+    
+    y_values = []
+    if (type == 'new'):
```

Comment:
One minor thing- python doesn't require these parentheses for if-statements
Usually our code will not have them